owl/EASE-OBJ.owl

```diff
@@ -2595,6 +2615,15 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Size -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Size">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Intrinsic"/>
+        <rdfs:comment>Size is the magnitude or dimension of a thing. Size can be measured as length, width, height, diameter, perimeter, area, volume.</rdfs:comment>
```

Comment:
Do not start by repeating the concept name. Just remove "Size is" from the first sentence.